api/helpers/game-states/unpack-gamestate.js

```diff
@@ -64,6 +64,7 @@ module.exports = {
       };
 
       const data = {
+        id: gameStateRow.id,
```

Comment:
Added the id of from the packed GameStateRow obj onto the unpacked GameState obj so that we could compare 'which' gameState we're talking about in `create-socket-event`. This lets us identify the correct squence of gamestates to check when counting consecutive passes